docs/api.md

```diff
@@ -695,6 +733,16 @@ Remove an image. Similar to the `docker rmi` command.
 * force (bool): Force removal of the image
 * noprune (bool): Do not delete untagged parents
 
+## remove_volume
+
+Remove a volume. Similar to the `docker volume rm` command.
+
+**Params**:
+
+* name (str): The volume's name
+
+**Returns** (bool): True on successful removal. Failure will raise an exception.
```

Comment:
It would be good to document which exception.

Using `rst` and sphinx you can do ``` `:class:docker.errors.APIError` ``` and get it all linked up :smiling_imp: